ansible/vars-bvt.yaml

```diff
@@ -12,6 +12,6 @@ avocado_conf: "{{avocado_conf_dir}}/avocado.conf"
 test_providers_dir: "{{avocado_data_dir}}/avocado-vt/test-providers.d/downloads"
 
 avocado_repos:
-  - { repo: "https://github.com/autotest/tp-libvirt.git", dest: "{{test_providers_dir}}/io-github-autotest-libvirt" }
-  - { repo: "https://github.com/autotest/tp-qemu.git", dest: "{{test_providers_dir}}/io-github-autotest-qemu" }
-  - { repo: "https://github.com/spiceqa/tp-spice.git", dest: "{{test_providers_dir}}/io-github-spiceqa-spice" }
+  - { repo: "https://{{pipeline_constants.GITHUB_DOMAIN}}/autotest/tp-libvirt.git", dest: "{{test_providers_dir}}/io-github-autotest-libvirt" }
+  - { repo: "https://{{pipeline_constants.GITHUB_DOMAIN}}/autotest/tp-qemu.git", dest: "{{test_providers_dir}}/io-github-autotest-qemu" }
+  - { repo: "https://{{pipeline_constants.GITHUB_DOMAIN}}/spiceqa/tp-spice.git", dest: "{{test_providers_dir}}/io-github-spiceqa-spice" }
```

Comment:
Those are upstream projects and repositories that we do not modify. They should keep the fixed "github.com" domain.